librz/util/ht/ht_uu.c

```diff
@@ -7,16 +7,10 @@
 
 #include "ht_inc.c"
 
-RZ_API HtName_(Ht) * Ht_(new0)(void) {
-	HT_(Options)
-	opt = {
-		.cmp = NULL,
-		.hashfn = NULL,
-		.dupkey = NULL,
-		.dupvalue = NULL,
-		.calcsizeK = NULL,
-		.calcsizeV = NULL,
-		.freefn = NULL
-	};
+/**
+ * \brief Create a new hashtable
```

Comment:
Also clarify, that nothing is set up and what needs to be set afterwards, so it works.